app/controllers/album.js

```diff
@@ -0,0 +1,30 @@
+const albumsService = require('../services/albums'),
+  logger = require('../logger'),
+  message = require('../constants');
+
+exports.getAlbumsInventory = (req, res, next) => {
+  logger.info(`${message.PREVIOUS_MESSAGE} to list of albums`);
+  albumsService
```

Comment:
Return promise

Reply:
We always have to put a `return` when we call a promise, to tell Node to "wait" for the promise to resolve/reject and not to get out of the currently function. If we don't put the `return` the endpoint will finish not considering the result of the promise we called